modular_zubbers/code/modules/customization/species/proteans/protean_modsuit.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/mod/control/pre_equipped/protean
-	name = "protean hardsuit rig"
-	desc = "The hardsuit rig unit of a Protean, allowing them to retract into it, or to deploy a suit that protects against various environments."
+	name = "protean modsuit"
```

Comment:
usual format is MOD control unit, so protean MOD control unit?
Does this ever show with the theme override?